restlicsharpclient/restliclient/request/builder/CreateRequestBuilder.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+   Copyright (c) 2017 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+
+using restlicsharpdata.restlidata;
+using restlicsharpclient.restliclient.response;
+
+namespace restlicsharpclient.restliclient.request.builder
+{
+    /// <summary>
+    /// Builder for constructing Rest.li CREATE Requests.
+    /// </summary>
+    /// <typeparam name="TKey">The key (id) type of the entity being created</typeparam>
+    /// <typeparam name="TEntity">The type of entity being created</typeparam>
+    public class CreateRequestBuilder<TKey, TEntity> : SingleEntityRequestBuilder<CreateRequest<TKey, TEntity>, EntityResponse<TEntity>, TKey, TEntity>
```

Comment:
CreateRequestBuilder should not extend from SingleEntityRequestBuilder (which is for request needs to set ID)